loop/README.md

```diff
@@ -0,0 +1,54 @@
+# Tests for Loop
+
+##General Configuration
+
+It's highly recommended to use [Virtualenv](https://virtualenv.pypa.io/en/latest/)
+in order to have an isolated environment.
+
+From /loop
+
+1. `virtualenv venv`
+2. `source venv/bin/activate`
+3. `pip install -r requirements.txt`
+
+Verify that the `manifest.ini` file contains expected values since the tests
+rely on using it for assertions.
+
+## Configuration Check Tests
+
+These tests are to be run whenever a deployment of loop happens. They access
+the server via HTTP calls to health check and status end points. Then the
+results are compared to known expected values.
+
+The tests are using [pytest](http://pytest.org/latest/) so to run them do the
+following:
+
+`py.test -v -s --env=<environment> --server=version=<version> config-check/`
+
+Where `<environment>` is `staging` or `production` and `<version>` is the
+version of loop server that is deployed to the environment you are testing
+against.
+
+## E2E Tests
+
+Tests in this directory are designed to test UI features of Loop. They require
+the use of Firefox with Marionette support built-in, which is Firefox 46 and
+above.
+
+These tests are currently designed to be run manually. Development work was
+done on Mac OS-X El Capitan so the location of the browser in the configuration
+file will need to change.
+
+The tests are using [pytest](http://pytest.org/latest/) so to run them do the
+following:
+
+`py.test -v --env=<environment> e2e-test/`
+
+where `<environemt>` is one of `stage` or `prod`.
+
+The tests start up a Firefox browser and run some automated tests. There might
```

Comment:
This seems a bit incomplete to have an automated test that requires user intervention in-between steps.  Probably wouldn't be too difficult to add the additional step in Marionette to accomplish this, but let me know if i can help

![loopclient](https://cloud.githubusercontent.com/assets/3699494/16435983/10c7d09c-3d4f-11e6-8c8f-d5e80712d82f.png)